x-pack/plugins/observability_ai_assistant/public/routes/conversations/conversation_view.tsx

```diff
@@ -111,11 +112,15 @@ export function ConversationView() {
     );
   }
 
+  function handleRefreshConversations() {
+    conversations.refresh();
+  }
+
   return (
     <>
       {confirmDeleteElement}
       <EuiFlexGroup direction="row" className={containerClassName}>
-        <EuiFlexItem grow={false}>
+        <EuiFlexItem grow={false} style={{ minWidth: 300, maxWidth: 300 }}>
```

Comment:
what's different here than `{{ width: 300 }}`?

Reply:
This may not be needed as adding textWrap to the items within in already made sure the sidebar was no longer getting pushed out. I'll check